adt_press/nodes/pdf_nodes.py

```diff
@@ -258,3 +259,23 @@ async def extract_metadata() -> BookMetadata:
 
 def book_table_of_contents(book_metadata: BookMetadata) -> list[BookChapter]:
     return book_metadata.table_of_contents
+
+
+def input_language(input_language_config: str | None, book_metadata: BookMetadata) -> Language:
```

Comment:
Backdown logic is now captured one level up (note `_config` nodes still exist to let OmegaConf resolve normally)